dipy/reconst/ivim.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/python
+""" Classes and functions for fitting ivim model """
+from __future__ import division, print_function, absolute_import
+import warnings
+import functools
+import numpy as np
+from dipy.core.optimize import Optimizer
+
+from scipy.optimize import leastsq
+
+from dipy.core.gradients import gradient_table
+from dipy.reconst.base import ReconstModel
+from dipy.reconst.dti import _min_positive_signal, apparent_diffusion_coef
+from dipy.reconst.dti import TensorModel, mean_diffusivity
+from .vec_val_sum import vec_val_vect
+from dipy.core.sphere import Sphere
+
+
+def ivim_function(params, bvals):
+    """The ivim function
+    Parameters
+    ----------
+    params : array
+             The IVIM parameters [S0, f, D_star, D]
+
+    bvals  : array
+             bvalues
+    """
+    S0, f, D_star, D = params
+    S = S0 * (f * np.exp(-bvals * D_star) + (1 - f) * np.exp(-bvals * D))
+    return S
+
+
+def _ivim_error(params, bvals, signal):
+    """Error function to be used in fitting the model
+    """
+    return (signal - ivim_function(params, bvals))
+
+
+class IvimModel(ReconstModel):
+    """Ivim model
+    """
+
+    def __init__(self, gtab, split_b=200.0, min_signal=None):
+        """
+        An IVIM model
+
+        Parameters
+        ----------
+        gtab : GradientTable class instance
+        split_b : split the data at this b-value for a two stage fit
```

Comment:
This should be: 

    split_b : float
        The b-value to split the data on for two-stage fit.